main.js

```diff
@@ -132,3 +132,8 @@ autoUpdater.on('update-downloaded', (info) => {
 app.on('ready', function()  {
   autoUpdater.checkForUpdates();
 });
+
+// Update will be downloaded now but installed only on the next launch of the app
+// app.on('ready', function()  {
+//   autoUpdater.checkForUpdatesAndNotify()
```

Comment:
I haven't tried a newer version with the function available.  What does it do?

Reply:
as mentioned in comment it will download the updated file but install only on the "exit/quit" on application. next time when user opens the app he'll get latest version.